MAUI/Cartesian-Charts/DataLabels.md

```diff
@@ -146,3 +146,123 @@ chart.Series.Add(series);
 {% endtabs %}
 
 ![Applying series interior for data label in MAUI chart](DataLabel_images/maui_chart_data_label_with_series_brush.jpg)
+
+## Formatting Label Context
+
+The content of the label can be customized using the [LabelContext]() property. Following are the two options that are supported now,
+
+* [Percentage]() - This will show the percentage value of corresponding data point Y value
```

Comment:
* [Percentage]() - This will show the percentage value of corresponding data point Y value.

Reply:
content updated